<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Model</class>
 <widget class="QMainWindow" name="Model">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Group by folder</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Group by type</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="displayBox">
          <item>
           <property name="text">
            <string>Table</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pie</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QStackedWidget" name="stackedWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
